rpi-uvc-gadget.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+# Variables we need to make things easier later on.
+
+CONFIGFS="/sys/kernel/config"
+GADGET="$CONFIGFS/usb_gadget"
+VID="0x0525"
+PID="0xa4a2"
+SERIAL="0123456789"
+MANUF=$(hostname)
+PRODUCT="UVC Gadget"
```

Comment:
Updating this to the same IDs and names as the original PiSight.

```suggestion
VID="0x05ac"
PID="0xdead"
DEVICE="0x0001"
SERIAL="1"
MANUF="Apple"
PRODUCT="PiSight"
```

Reply:
Not no nitpick, but if we call this a PiSight, is it really still made by Apple? And conversely, if we say it's made by Apple, shouldn't we call it iSight?

I'd say hide an Easter egg in there

Reply:
I was thinking that it makes most sense if this PR is just focused on restoring functionality without changing other aspects of the product. We can then still have separate branches or PRs for different ways that the camera shows up.